### ISSUE TITLE
Add some explanations in Wiki for Jquery import

### ISSUE THREAD

**Issue by phiphou** [AngularClass/angular2-webpack-starter#723]
Hi,

If we need to include [jQuery](https://jquery.com/), instructions given in the [wiki](https://github.com/AngularClass/angular2-webpack-starter/wiki/How-to-include-jQuery) work fine, but **there are some special cases** we need to be aware of.

Let me explain with a simple use case : **adding [Foundation-sites](http://foundation.zurb.com/sites.html) in the project.**

Foundation-sites has a dependency to jQuery, so when installed it will install jQuery in our `node_modules` folder. But in the Foundation-site module, there's also a `vendor `folder that provide JQuery too (**and not necessary the same** than the one listed in the `package.json`'s dependencies of the Foundation-site module).

So, if we follow only your instructions, for this use case, **these two versions of jQuery will be imported** in the generated `vendor.js` bundle.

We can handle these kind of thing with a **[null-loader](https://github.com/webpack/null-loader) webpack loader** for things we want to be sure they'll not be imported.

Something like that  :
```javascript
module: {
  loaders: [{
    test: /jquery\.js/,
    loader: 'null-loader',
    exclude: path.resolve('node_modules/foundation-sites/')
  }]
}
```
Obviously, only the `webpack.ProvidePlugin` method is concerned. Maybe add some other methods (like with [script-loader](https://github.com/webpack/script-loader) for instance) could also be useful, don't you think?

Don't you think these informations could be useful for others? If so, maybe you should add them in the [Wiki](https://github.com/AngularClass/angular2-webpack-starter/wiki)?

**Comment by gdi2290** [comment]
ya, can you update the wiki to mention this. Thanks!

**Comment by phiphou** [comment]
I realized that I don't identified the reason well. 

If there is no jquery module as "root module" (in the root of the main node_modules folder), and you install foundation-sites, this jquery module will be installed as "root module". And your solution works well. 

But if another module depends on jQuery, and if it is not the same than the previous one,  when installed, the "previous one" will be copied in the foundation-sites's node_modules  directory, and the "new one" as "root module".

In this cas, we need the solution I provided. 

Let me do other tests, clean everything and I'll update the wiki ;)
